### PR TITLE
Merge apex-runner to master

### PR DESCRIPTION
**Pull request by tweise**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---
R: @kennknowles

### CODE REVIEW

**Comment by kennknowles** [comment]
I don't believe the one Travis failure is related. It is a failure of the Include What You Use analysis for an archetype.

**Comment by kennknowles** [comment]
There are conflicts now, so let's re-merge master into this PR and resolve the conflicts.

**Comment by tweise** [comment]
Resolved the conflict.

File: examples/java/pom.xml [hunk 1]
```diff
@@ -223,6 +235,29 @@
...
               </execution>
+              <execution>
+                <id>apex-runner-integration-tests</id>
+                <goals>
+                  <goal>integration-test</goal>
+                  <goal>verify</goal>
+                </goals>
+                <configuration>
+                  <includes>
+                    <include>WordCountIT.java</include>
```
Comment (kennknowles) [inline examples/java/pom.xml]: Non-blocking: filed [BEAM-966](https://issues.apache.org/jira/browse/BEAM-966). There are matching issues for Flink runner and Spark runner.

**Comment by kennknowles** [comment]
Confirmed that the expected integration tests and unit tests ran. LGTM and nice to have a green, clean, merge.

**Pull request merged**